src/multi_vector_simulator/C0_data_processing.py

```diff
@@ -1530,10 +1530,47 @@ def receive_timeseries_from_csv(
         compute_timeseries_properties(dict_asset)
 
 
+def replace_nans_in_timeseries_with_0(timeseries, label):
+    """
+
+    Replaces nans in the timeseries (if any) with 0
+
+    Parameters
+
+    ----------
+    timeseries: pd.Series
+        timeseries in dict_asset (having nan value(s) if any), also of parameters
+
+    label: str
+        Key to print into the error message
+
+    Returns
+    ----------
+    timeseries: pd.Series
+        timeseries without NaN values
+
+    Notes
+    -----
+    Function tested with
+    - C0.test_replace_nans_in_timeseries_with_0()
+    """
+
+    if (pd.isna(timeseries)).any():
+        incidents = sum(pd.isna(timeseries))
+        logging.info(str(timeseries.values))
```

Comment:
Wouldn't it clog the log file otherwise?
```suggestion
        logging.debug(str(timeseries.values))
```

Reply:
Uups, I should remove that line actually.